test/WebJobs.Script.Tests/ContainerManagement/LinuxContainerInitializationHostServiceTests.cs

```diff
@@ -0,0 +1,145 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Script.Config;
+using Microsoft.Azure.WebJobs.Script.WebHost.ContainerManagement;
+using Microsoft.Azure.WebJobs.Script.WebHost.Management;
+using Microsoft.Azure.WebJobs.Script.WebHost.Models;
+using Microsoft.Azure.WebJobs.Script.WebHost.Security;
+using Microsoft.Extensions.Logging.Abstractions;
+using Moq;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests.ContainerManagement
+{
+    public class LinuxContainerInitializationHostServiceTests : IDisposable
```

Comment:
Look at the [Trait] attributes applied to InstanceManagerTests. These attributes serve to isolate the instance manager / assign tests (see run-tests.ps1 in the repo root) to reduce test flakiness. I think your new tests should run in the same group. Add a new TestTrait property (see TestTraits.StandbyModeTests), e.g. ContainerInstanceTests and put both test suites in that same group.